models/br_bd_diretorios_mx/code/upload.py

```
@@ -0,0 +1,101 @@
+        path=str(path),
+        source_format="parquet",
+        if_table_exists="replace",
+        if_storage_data_exists="replace",
+        if_dataset_exists="pass",
+    )
+    try:
+        c = bigquery.Client(project=BILLING)
+        n = next(
+            iter(
+                c.query(
+                    f"select count(*) n from `{BILLING}.{DATASET_ID}_staging.{slug}`"
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)